optaweb-employee-rostering-frontend/cypress/integration/newTenantBasicWorkflow.js

```diff
@@ -77,7 +77,7 @@ describe('A new tenant can be created, who can have their own employees, spots,
     cy.get('[data-cy=settings]').click();
     cy.get('[data-cy=add-tenant]').click();
     cy.get('[data-cy=name]').type('Test Tenant');
-    cy.get('[data-cy=schedule-start-date]').type('2018-01-01');
+    cy.get('[data-cy=schedule-start-date]').type(Cypress.moment().format("YYYY-MM-DD"));
```

Comment:
I dislike not using a static date. I would prefer using https://docs.cypress.io/api/commands/clock.html#No-Args to change the current date.

Reply:
No problem, could you only specify why the clock is more appropriate here since I was feeling that clock is just for time not a date.

Reply:
See https://docs.cypress.io/api/commands/clock.html#Now . JavaScript treats dates as time, time as dates.